sections/api/typescript.md

```diff
@@ -178,6 +178,49 @@ This is the most complex example where we have specific properties for the styli
 the rest of the required properties by the **Header** through. That means when you use **Title** it will have
 the combined typing of both the styled requirements and the actual component requirements.
 
+### Using `attrs` method
+
+The `attrs` method accepts an Interpolation object, which can be either a string that will be merged into the rest of the component's props, or a function that receives the styled component's `props` as the first and only argument.
```

Comment:
The description does not seem to be accurate to me. The `attrs` method does not interpolate (using backtick or a function). It passes set of predefined prop values down to the decorated component. So the examples seem right but not the description.

I'd suggest to describe it similarly to the website:

```suggestion
This is a chainable method that attaches some props to a styled component. It accepts an object with partial props that will be merged into the rest of the component's props. Alternatively, a function can be passed that takes interior props and returns an object to merge.
```